cmd/crc/cmd/daemon.go

```diff
@@ -27,6 +27,8 @@ func init() {
 	rootCmd.AddCommand(daemonCmd)
 }
 
+const hostVirtualIP = "192.168.127.254"
```

Comment:
we might want to have these in a better spot.

Reply:
yes